model_analyzer/analyzer.py

```diff
@@ -212,8 +213,14 @@ def _create_model_manager(self, client, gpus):
             metrics_manager=self._metrics_manager,
             state_manager=self._state_manager)
 
-    def _get_server_only_metrics(self, client):
+    def _get_server_only_metrics(self, client, gpus):
         if self._config.triton_launch_mode != 'c_api':
```

Comment:
Does this work for remote mode? Do the GPUs actually match what is on the remote machine?